spec/javascripts/tests/Header_spec.js

```diff
@@ -0,0 +1,65 @@
+describe('Header', function() {
+  'use strict';
+
+  beforeEach(function(done) {
+    var self = this;
+
+    fixture.setBase('spec/javascripts/fixtures');
+
+    requirejs(
+      ['jquery', 'Header'],
+      function($, Header) {
+        fixture.load('Header.html');
+
+        self.$header = $(fixture.el).find('[data-dough-component="Header"]');
+        self.obj = new Header(self.$header);
+
+
+        self.activeClass = 'mobile-header__search-active';
+        self.$header = $(document).find('[data-site-header]');
```

Comment:
This is defined twice (first one is OK).

Reply:
12f94cd